.release/release-metadata.hcl

```diff
@@ -0,0 +1,4 @@
+url_docker_registry_dockerhub = "https://hub.docker.com/r/hashicorp/consul-k8s"
```

Comment:
@mdeggies we use registry today for the docker images: https://hub.docker.com/r/hashicorp/consul-k8s-control-plane/tags